src/presets/erc20.cairo

```diff
@@ -31,6 +31,22 @@ mod ERC20 {
         ERC20Event: ERC20Component::Event
     }
 
+    impl ERC20VotesHooksImpl<TContractState> of ERC20Component::ERC20HooksTrait<TContractState> {
+        fn before_update(
+            ref self: ERC20Component::ComponentState<TContractState>,
+            from: ContractAddress,
+            recipient: ContractAddress,
+            amount: u256
+        ) {}
+
+        fn after_update(
+            ref self: ERC20Component::ComponentState<TContractState>,
+            from: ContractAddress,
+            recipient: ContractAddress,
+            amount: u256
+        ) {}
+    }
```

Comment:
I tried, but the issue is that the compiler won't let us define an implementation in the ERC20Component module, or the file, because it conflicts with the generic expected implementation. We could provide an implementation outside the module, but I couldn't come up with a design I really liked, and I rather the user defining the implementation of the hooks anyway, even if is empty, so they are explicitly aware of what is happening after and before update (that could be problematic)

Reply:
Updated!